mlflow/recipes/utils/metrics.py

```diff
@@ -46,6 +46,20 @@ def from_custom_metric_dict(cls, custom_metric_dict):
     RecipeMetric(name="precision_score", greater_is_better=True),
     RecipeMetric(name="f1_score", greater_is_better=True),
     RecipeMetric(name="accuracy_score", greater_is_better=True),
+    RecipeMetric(name="roc_auc", greater_is_better=True),
+]
+
+BUILTIN_MULTICLASS_CLASSIFICATION_RECIPE_METRICS = [
+    RecipeMetric(name="true_negatives", greater_is_better=True),
+    RecipeMetric(name="false_positives", greater_is_better=False),
+    RecipeMetric(name="false_negatives", greater_is_better=False),
+    RecipeMetric(name="true_positives", greater_is_better=True),
```

Comment:
remove those metrics, which are for per-class not the aggregated metric

Reply:
Done!